modules/ovms_ai_extension/src/ams_wrapper/requirements.txt

```diff
@@ -0,0 +1,6 @@
+tensorflow-serving-api==2.7.0
```

Comment:
There's a lightweight Python client for OVMS available. See:
- https://pypi.org/project/ovmsclient/
- https://community.intel.com/t5/Blogs/Tech-Innovation/Artificial-Intelligence-AI/Simplified-Deployments-with-OpenVINO-Model-Server-and-TensorFlow/post/1353218

Switching to `ovmsclient` will let you significantly reduce overall Docker image size. But this gain is possible only, if you drop tensorflow-serving-api and all it's dependencies (including tensorflow itself).